mne/epochs.py

```diff
@@ -2640,6 +2696,134 @@ def _get_epoch_from_raw(self, idx, verbose=None):
                                             self.reject_by_annotation)
         return data
 
+    @property
+    def raw_annotations(self):  # noqa: D102
```

Comment:
It was named this because `concatenate_epochs` checks for an attribute called `annotations`, so then a bunch of warnings were raised.

Instead, I added a line to `conftest.py` to ignore this warning for now.